### ISSUE TITLE
android-v6.0.0 - Could not find com.mapbox.mapboxsdk:mapbox-android-core:0.3.0-SNAPSHOT

### ISSUE THREAD

**Issue by mbrame12** [mapbox/mapbox-gl-native#11708]
**Mapbox SDK version:**6.0.0

### Steps to trigger behavior

 1. I saw that version 6.0.0 of the Android SDK was released, so I changed the following lines in my build.gradle file from:

```
    implementation('com.mapbox.mapboxsdk:mapbox-android-sdk:6.0.0-beta.7') {
        transitive = true
    }
```

to 

```
    implementation('com.mapbox.mapboxsdk:mapbox-android-sdk:6.0.0') {
        transitive = true
    }
```
and I got a message about being unable to resolve all dependencies:

> Could not find com.mapbox.mapboxsdk:mapbox-android-core:0.3.0-SNAPSHOT

**Comment by tobrun** [comment]
It seems that upstream dependency in https://github.com/mapbox/mapbox-events-android shipped a patch version with a SNAPSHOT, to work around this you need to add the following lines to:

```java
allprojects {
    repositories {
        jcenter()
        maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
    }
}
```

**Comment by LukasPaczos** [comment]
Fix landed with #11710.
